test/puppetlabs/trapperkeeper/services_test.clj

```diff
@@ -62,6 +65,50 @@
           app (bootstrap-services-with-empty-config [service1])]
       (is (not (nil? app))))))
 
+(deftest test-restart-file-correct-count
+  (testing "create a restart file and check that it correctly increments when a service starts"
+    (let [service1 (service Service1
+                            []
+                            (service1-fn [this] "hi"))
+          service2 (service Service1
+                            []
+                            (service1-fn [this] "hi"))]
+          (with-app-with-config app [service1] {:global {:restart-file "./dev-resources/counter"}}
+            (is (some? app))
```

Comment:
I was thinking of actually taking that line completely out since it is tested in the above test and is not relevant to this test